### PR TITLE
Association users ideas

### PR DESCRIPTION
**Pull request by korneliacieslik**

### CODE REVIEW

File: app/controllers/ideas_controller.rb
Comment (nidalaa) [inline app/controllers/ideas_controller.rb]: ❤️

File: app/models/idea.rb
Comment (nidalaa) [inline app/models/idea.rb]: Indentation looks a bit strange, is it two spaces or two tabs?
Try to change your settings in Sublime https://stackoverflow.com/a/22704309

File: db/schema.rb
Comment (nidalaa) [inline db/schema.rb]: 👏  for index! 🎉

File: app/controllers/ideas_controller.rb
Comment (nidalaa) [inline app/controllers/ideas_controller.rb]: I's be a bit more explicit with current user and use 

current_user.ideas.new (idea params) in `new` action 🤔 .

However, there is nothing wrong with this approach, so take it or leave it :)

**Review by nidalaa — CHANGES_REQUESTED** [review]

File: app/controllers/ideas_controller.rb
Comment (wmaciejak) [reply app/controllers/ideas_controller.rb]: Really good point! @korneliacieslik would you mind to take a look ? :)

Comment (korneliacieslik) [reply app/controllers/ideas_controller.rb]: @nidalaa  it is fixed now. Could you take a look?

File: app/models/idea.rb
Comment (korneliacieslik) [reply app/models/idea.rb]: It's fixed

**Comment by nidalaa** [comment]
@korneliacieslik LGTM, ready to merge once you resolve conflicts 👍 
Please hit Merge button once it is done! I tryst you here! :shipit:

**Pull request merged**
